### PR TITLE
Add Connect and Query Kuzzle function documentation

### PR DESCRIPTION
**Pull request by y-abs**
## What does this PR do ?

Add documentation for `query` and `connect` Kuzzle function

### CODE REVIEW

File: doc/2/core-classes/kuzzle/query/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+code: true
+type: page
+title: Query
```
Comment (Aschen) [inline doc/2/core-classes/kuzzle/query/index.md]:
```suggestion
title: query
```

File: doc/2/core-classes/kuzzle/query/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+description: Base method to send API query to Kuzzle
+---
+
+# query
+
+Base method used to send queries to Kuzzle, following the [API Documentation](/core/2/api).
+
+:::warning
```
Comment (Aschen) [inline doc/2/core-classes/kuzzle/query/index.md]:
```suggestion
::: warning
```

File: doc/2/core-classes/kuzzle/query/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      {Map<String, dynamic> volatile, bool queueable = true})
+```
+
+<br/>
+
+| Argument  | Type              | Description            |
+| --------- | ----------------- | ---------------------- |
+| `request` | <pre>KuzzleRequest</pre> | API request    |
+| `volatile`   | <pre>Map<String, dynamic></pre> | Additional information to send to Kuzzle |
+| `queueable`   | <pre>bool</pre> | Additional information to send to Kuzzle |
```
Comment (Aschen) [inline doc/2/core-classes/kuzzle/query/index.md]: Wrong description

File: doc/2/core-classes/kuzzle/connect/index.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+code: true
+type: page
+title: Connect
```
Comment (Aschen) [inline doc/2/core-classes/kuzzle/connect/index.md]:
```suggestion
title: connect
```

File: doc/2/core-classes/kuzzle/connect/index.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+code: true
+type: page
```
Comment (Aschen) [inline doc/2/core-classes/kuzzle/connect/index.md:3]:
```suggestion
type: code
```

File: doc/2/core-classes/kuzzle/connect/index.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+title: Connect
+description: Connects the SDK to Kuzzle
+---
+
+# Connect
```
Comment (Aschen) [inline doc/2/core-classes/kuzzle/connect/index.md]:
```suggestion
# connect
```

File: doc/2/core-classes/kuzzle/query/index.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
+---
+code: true
+type: page
```
Comment (Aschen) [inline doc/2/core-classes/kuzzle/query/index.md:3]:
```suggestion
type: code
```

**Review by Aschen — CHANGES_REQUESTED** [review]

**Review by scottinet — APPROVED** [review]

**Review by Aschen — APPROVED** [review]

**Pull request merged**